src/libmongoc/src/mongoc/mongoc-util-private.h

```diff
@@ -255,6 +255,13 @@ typedef struct {
    uint64_t value;
 } mcd_optional_u64_t;
 
+/**
+ * Returns true if the Windows version is greater than or equal to the required
+ * desktop or server version.
+ */
+bool
+_mongoc_verify_windows_version(int major_version, int minor_version, int build_number, bool strictly_equal);
```

Comment:
Suggest using `DWORD` to match type expected by [OSVERSIONINFOEXW](https://learn.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-osversioninfoexw):

```c
#ifdef _WIN32
bool
_mongoc_verify_windows_version(DWORD major_version, DWORD minor_version, DWORD build_number, bool strictly_equal);
#endif
```